logging/tests/unit/handlers/test__helpers.py

```diff
@@ -63,6 +64,57 @@ def test_valid_context_header(self):
         self.assertEqual(trace_id, expected_trace_id)
 
 
+# webapp2 has not been ported to python3 yet, so do not try to test its
+# functionality there.
+if sys.version_info[0] == 2:
+    class Test_get_trace_id_from_webapp2(unittest.TestCase):
+
+        @staticmethod
+        def create_app():
+            import webapp2
+
+            class GetTraceId(webapp2.RequestHandler):
+                def get(self):
+                    from google.cloud.logging.handlers import _helpers
+
+                    trace_id = _helpers.get_trace_id_from_webapp2()
+                    self.response.content_type = "text/plain"
+                    self.response.out.write(trace_id)
+
+            app = webapp2.WSGIApplication([
+                ('/', GetTraceId),
+            ])
+
+            return app
+
+        def setUp(self):
+            self.app = self.create_app()
+
+        def test_no_context_header(self):
+            import webob
+
+            req = webob.BaseRequest.blank('/')
+            response = req.get_response(self.app)
+            trace_id = response.body
+
+            self.assertEquals("None", trace_id)
```

Comment:
(1) is just a side-effect of how I wrote the test: I had the handler return the trace-id as text.  If the trace-id is None, then it gets auto-stringified to `None`.  I don't think webapp2 has something like flask's `test_request_context` so we have to go through a string intermediary at some point.

I could return json instead of just a raw text string, and then the json would convert `null` back to `None` -- would that be better?